### PR TITLE
Style settings page header

### PR DESCRIPTION
**Pull request by yscik**
Fixes 163-gh-Automattic/wpjm-addons

Also moved the Save button to the header, because it can be hard to find/remember sometimes when the page is long.
+ Added the styled header to the Marketplace page too.

### Changes Proposed in this Pull Request

* Add a header with logo to the Settings and Marketplace changes
* Tweak some colors and spacing, set max width
* Move the Save Changes button to the header, make it change color when there are changes to save
* Fix some scroll jumping
* Fix search in the Marketplace page going to the old `..-addons` URL
* Update the outdated lost license link in the License page

### Testing Instructions

* Open the Job Manager → Settings page, click around, change some settings, make sure everything works right
* Also check out the Job Manager → Marketplace page

### Release Notes

* Update design for settings and marketplace pages

### Screenshot / Video

image

image

----

| Plugin build for 0b81e90db5918d4c4e3e4f0f96434fd3aa73fa73 <a href="#"></a> |
| ------------------------------------------------------------ |
| 📦 [Download plugin zip](https://wpjobmanager.com/wp-content/uploads/2023/11/wp-job-manager-zip-2641-0b81e90d.zip)                       |
| ▶️ [Open in playground](https://wpjobmanager.com/playground/?core=2023/11/2641-0b81e90d)             |

### CODE REVIEW

**Comment by masteradhoc** [comment]
Looks great @yscik!

File: includes/admin/class-wp-job-manager-addons.php [hunk 1]
```diff
@@ -180,53 +180,65 @@ public function get_animated_logo() {
...
-				</a>
-				<?php if ( current_user_can( 'update_plugins' ) ) : ?>
-					<a href="<?php echo esc_url( admin_url( 'edit.php?post_type=job_listing&page=job-manager-marketplace&section=helper' ) ); ?>"
-						class="nav-tab
+		<div class="wrap wp_job_manager wp_job_manager_add_ons_wrap job-manager-settings-wrap">
+			<div class="job-manager-settings-header-wrap">
+				<div class="job-manager-settings-header">
+					<div class="job-manager-settings-header-row">
+						<img class="job-manager-settings-logo"
+							src="<?php echo esc_attr( JOB_MANAGER_PLUGIN_URL . '/assets/images/jm-full-logo.png' ); ?>"
```
Comment (mikeyarce) [inline includes/admin/class-wp-job-manager-addons.php]: This should be `esc_url()` here.

File: includes/admin/class-wp-job-manager-settings.php [hunk 2]
```diff
@@ -552,13 +552,26 @@ public function output() {
...
-					foreach ( $this->settings as $key => $section ) {
-						echo '<a href="#settings-' . esc_attr( sanitize_title( $key ) ) . '" class="nav-tab">' . esc_html( $section[0] ) . '</a>';
-					}
-					?>
-				</h2>
+				<div class="job-manager-settings-header-wrap">
+					<div class="job-manager-settings-header">
+						<div class="job-manager-settings-header-row">
+							<img class="job-manager-settings-logo"
+								src="<?php echo esc_attr( JOB_MANAGER_PLUGIN_URL . '/assets/images/jm-full-logo.png' ); ?>"
```
Comment (mikeyarce) [inline includes/admin/class-wp-job-manager-settings.php]: Should be `esc_url()` here.

File: includes/admin/class-wp-job-manager-settings.php [hunk 3]
```diff
@@ -552,13 +552,26 @@ public function output() {
...
+								alt="<?php esc_attr_e( 'Job Manager', 'wp-job-manager' ); ?>" />
+							<input type="submit" class="job-manager-settings-submit wpjm-button is-outline" value="<?php esc_attr_e( 'Save Changes', 'wp-job-manager' ); ?>" />
+						</div>
+
+						<div class="nav-tab-wrapper">
+							<?php
+							foreach ( $this->settings as $key => $section ) {
+								echo '<a href="#settings-' . esc_attr( sanitize_title( $key ) ) . '" class="nav-tab">' . esc_html( $section[0] ) . '</a>';
```
Comment (mikeyarce) [inline includes/admin/class-wp-job-manager-settings.php]: `esc_url()` here, too.  Probably best to wrap the whole URL inside `esc_url()` though.  

Something like
```
echo '<a href="' . esc_url( '#settings-' . sanitize_title( $key ) ) . '" class="nav-tab">' . esc_html( $section[0] ) . '</a>';
```

**Review by mikeyarce — CHANGES_REQUESTED** [review]
Looks great @yscik !

Added a few small escaping things.

I also found one small issue - which I think are actually not new but might be good to address?

When in a mobile view, if you click on something like "Job Submission" - there's a `margin-bottom: 1px` which is pushing the other elements below it:
CleanShot 2023-11-17 at 11 07 29@2x

This CSS seems to fix it:

```
.nav-tab.nav-tab-active {
  margin-bottom: 0;
}
```

**Comment by yscik** [comment]
Thanks! Improved the escaping and did some fixes for mobile, with smaller logo, non-sticky header and a save button in the bottom.

image

**Review by mikeyarce — APPROVED** [review]
Looks really good! 🎉

**Pull request merged**